infrastructure/src/main/java/org/corfudb/infrastructure/ServerContext.java

```diff
@@ -482,9 +482,13 @@ public synchronized void saveManagementLayout(Layout layout) {
             dataStore.put(Layout.class, PREFIX_MANAGEMENT, MANAGEMENT_LAYOUT, layout);
             log.info("saveManagementLayout: Updated to new layout at epoch {}",
                     getManagementLayout().getEpoch());
-        } else {
+        } else if (layout.getEpoch() < currentLayout.getEpoch()){
```

Comment:
I'm differentiating between layout.getEpock < currentLayout.getEpoch() and layout.getEpock == currentLayout.getEpoch() as the second one is very noisy. thought? we see a lot of these:
corfu.9000.1.log.gz:2019-01-31T13:35:25.366Z | DEBUG |              ManagementService |            o.c.i.ServerContext | saveManagementLayout: Ignoring layout because new epoch 4 <= old epoch 4
corfu.9000.1.log.gz:2019-01-31T13:35:26.366Z | DEBUG |              ManagementService |            o.c.i.ServerContext | saveManagementLayout: Ignoring layout because new epoch 4 <= old epoch 4
corfu.9000.1.log.gz:2019-01-31T13:35:27.366Z | DEBUG |              ManagementService |            o.c.i.ServerContext | saveManagementLayout: Ignoring layout because new epoch 4 <= old epoch 4

Reply:
ok. Done as we discussed offline.